kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/FieldNameSanitizer.java

```diff
@@ -0,0 +1,31 @@
+package com.wepay.kafka.connect.bigquery.utils;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FieldNameSanitizer {
+
+  private static String sanitizeName(String name) {
+    name = name.replaceAll("^[^a-zA-Z_]+", "");
```

Comment:
if it doesn't start with a letter or underscore, then may adding an underscore at the beginning would be preferable? It would preserve more of the original field name I think